picoscope/ps4000a.py

```diff
@@ -140,14 +146,14 @@ def __init__(self, serialNumber=None, connect=True):
 
         super(PS4000a, self).__init__(serialNumber, connect)
 
-    def _lowLevelOpenUnit(self, sn):
+    def _lowLevelOpenUnit(self, serial):
```

Comment:
can we keep this as "serialnumber" please? serial can mean "protocol" and for such low level functions (especially for an oscilloscope) it might be confused with that.

I do agree that `sn` is very bad practice.